### ISSUE TITLE
[Feature Suggestion] My Neat Feature

### ISSUE THREAD

**Issue by Demomute** [gta-chaos-mod/ChaosModV#1849]
Make it so everyone is a Ghost gives you a black arrow! Or make it so when everyone is a ghost everyone is blind (so the cops won't spot you etc)

**Comment by JustTunerYT** [comment]
I don't think it should be "Feature Request", as "Everyone is a ghost" is an effect, so it's "Effect Improvement Suggestion" or such.
But idea is good. That would add detail to it
